src/core/client/stream/tabs/Comments/Comment/ReactionButton/ReactionButtonContainer.tsx

```diff
@@ -126,7 +125,6 @@ const ReactionButtonContainer: FunctionComponent<Props> = ({
       readOnly={readOnly}
       isQA={isQA}
       author={comment.author?.username}
-      iconClassName={CLASSES.comment.actionBar.reactButton}
```

Comment:
I think we can remove the `iconClassName` prop from `ReactionButton` now since this was the only place it was being used.